### ISSUE TITLE
Ledger doesn't understand full absolute paths in "include" on Windows

### ISSUE THREAD

**Issue by ngleb** [ledger/ledger#1670]
Initially I though this was an org-babel problem, but actually it is not. Let's say I have a journal file with transactions `main.ledger` at `d:/ledger/main.ledger`.

Ledger is build using the latest msys2/mingw32 as described in this guide: [build ledger](https://gist.github.com/ngleb/944346a67960776371aab74149edf38c)

I want to include the file in org-babel like this:
```
!include d:/ledger/main.ledger
```
This won't work, ledger will say it can't read journal file. The path expansions works incorrectly. Then I checked the same case without org-babel or emacs at all, and the same problem occurred.

If I run command from command line like this, then it works:
```
ledger -f d:/ledger/main.ledger bal
```
If I add another file called `d:/ledger/test.ledger` with the following content:
```
!include d:/ledger/main.ledger
```
and then run it like this:
```
ledger -f d:/ledger/test.ledger bal
```

Then it will not work. The workaround for includes is easy: don't use absolute paths, and in case of org-babel you can alwasy specify the file via `:cmdline` option by adding `-f "path to file"`.

Can it be resolved at all? I'm not sure how it should work on Linux and I actually didn't check it yet.

**Comment by zpeters** [comment]
Have you tried "//" or "\" in place of "/"?

**Comment by ngleb** [comment]
> Have you tried "//" or "" in place of "/"?

yes, I checked different variants. The problem is with the starting symbol which isn't a relative path but a driver letter, so ledger could use it as an indicator.

**Comment by tbm** [comment]
same as #1188?

**Comment by ngleb** [comment]
I don't use cygwin, my build was made under msys/mingw32 which is a native binary for Windows.

I don't think it's the same issue. Mine was a problem with ```!include```, and the other ticket says that Cygwin ledger binary doesn't expand path properly when run from command line. mingw version doesn't have this problem.
